src/gbd_mapping_generator/util.py

```diff
@@ -8,7 +8,8 @@
 
 TEXTWIDTH = 118  # type: int
 TAB = "    "  # type: str
-SPACING = "\n\n"  # type: str
+SINGLE_SPACING = "\n"  # type: str
+DOUBLE_SPACING = "\n\n"  # type: str
```

Comment:
What's the story with single and double spacing? Was the double spacing breaking black in some places?

Also, can this not be specified as `DOUBLE_SPACING: str = "\n\n"`?

Reply:
Yeah the old spacing would break linting. I'll go ahead and update the type annotation